@stellar/lyra-api/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@stellar/lyra-api",
+  "main": "build/index.js",
+  "types": "build/@stellar/lyra-sdk/src/index.d.ts",
+  "prettier": "@stellar/prettier-config",
+  "version": "1.0.0-alpha",
+  "description": "Utility functions to interact with Lyra extension",
+  "repository": {
+    "url": "git@github.com:stellar/lyra.git",
+    "type": "git"
+  },
+  "author": "Stellar Development Foundation <hello@stellar.org>",
+  "browserslist": {
+    "production": [
+      "last 3 chrome version",
+      "last 3 firefox version"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version"
+    ]
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@lyra/api": "1.0.0",
```

Comment:
I think we might need to find a different way to consume this? Since this "package" comes from the extension, I'm not able to install this module from a packed tarball. It fails because `@lyra/api` doesn't exist on npm

Reply:
This should be fixed now!